### PR TITLE
[MRG] Provide gradients to EI

### PR DESCRIPTION
**Pull request by MechCoder**
Fixes https://github.com/scikit-optimize/scikit-optimize/issues/191

### CODE REVIEW

**Comment by MechCoder** [comment]
Running benchmarks.

**Comment by MechCoder** [comment]
The benchmarks aren't as great as before in terms of speed, but there is still some improvement.

```
Hart6:
In this branch:
Mean optimum: -3.04108840345
Std of optimal values0.160640717755
Best optima:-3.28395367839
Mean time: 50.7440725565
Best time:38.7790241241

In master:
Mean optimum: -3.05233467105
Std of optimal values0.11063339838
Best optima:-3.30817189775
Mean time: 63.0642666101
Best optima:53.6385185719

Branin:
In this branch:
Mean optimum: 0.399999558439
Std of optimal values0.00108553732474
Best optima:0.39827485431
Mean time: 39.6305116415
Best time:36.1627657413

In master:
Mean optimum: 0.406778209301
Std of optimal values0.00801767344005
Best optima:0.39811029196
Mean time: 44.0078294992
Best time:38.6263272762
```

**Comment by MechCoder** [comment]
So, now there are no lbfgs-based tests in the top 10 slowest.

@glouppe @betatim thoughts?

File: skopt/acquisition.py [hunk 1]
```diff
@@ -4,7 +4,19 @@
 from scipy.stats import norm
 
 
-def _gaussian_acquisition(X, model, y_opt=None, method="LCB",
+def acquisition_1D(X, model, y_opt=None, acq_func="LCB", xi=0.01,
```
Comment (glouppe) [inline skopt/acquisition.py]: `acquisition_1D` is not a good name, as it is wrapping `_gaussian_acquisition` only.

**Comment by glouppe** [comment]
Not much to comment, this looks good to me! Very good work :)

**Comment by MechCoder** [comment]
will merge when tests pass

**Pull request merged**